server/app.js

```diff
@@ -378,6 +374,14 @@ app.get('/account', asyncHandler(async (req, res, next)=>{
 	}
 
 	req.brew = data;
+
+	req.ogMeta = { ...defaultMetaTags,
+		title       : `${req.params.username}'s Account`,

```

Comment:
This isn't account information for a specific user, but rather the account information of the currently logged in user, so I think this title needs to be updated.

Reply:
Oh good catch. I just changed it to something generic, "Account Page".